xarizmi/candlestick.py

```diff
@@ -9,6 +9,26 @@
 
 
 class Candlestick(BaseModel):
+    """[Summary?]
+
+    :param close: the closing price for the corresponding time period
+    :type close: float
+    :param open: the opening price for the corresponding time period
+    :type open: float
+    :param low: the lowest traded price for the corresponding time period
+    :type low: float
+    :param high: the highest traded price for the corresponding time period
+    :type high: float
+    :param volume: [volume description?]
+    :type high: float
+    :param amount: [amount description?]
+    :type amount: float
+    :param interval_type: [interval_type description?]
+    :type interval_type: [interval_type type?]
+    :param symbol: [symbol description?]
+    :type symbol: [symbol type?]
```

Comment:
What is the symbol parameter used for? And what is its type?

Reply:
@rezaeb4di 
The symbol is string.

For example symbol can "BTC-USDT" where BTC stands for bitcoin and usdt for Teter.
Or it can be like "CAD-USD" where CAD is Canadiant dollar and USD is US Dollar.